env.yaml

```diff
@@ -2,19 +2,21 @@ channels:
   - conda-forge
   - bioconda
 dependencies:
-  - python >= 3.9
-  - pandas
+  - python
   - fastqc
   - samtools
   - bedtools
-  - pysam
   - pip
-  - requests
   - snakemake
   - porechop_abi
   - multiqc
   - kraken2
+  - ont-modkit
   - pip:
+    - pandas
```

Comment:
What's the reason to move pandas etc under pip ? I'd stick to keeping as much as possible under conda to solve.

Reply:
ok, I will  move it